examples/mnist/prepare_mnist_data_for_ludwig.py

```diff
@@ -1,13 +1,58 @@
-import os
-import os.path
-
-
-for name in ['training', 'testing']:
-    with open(os.path.join('data','mnist_dataset_{}.csv'.format(name)), 'w') as output_file:
-        print('=== creating {} dataset ==='.format(name))
-        output_file.write('image_path,label\n')
-        for i in range(10):
-            path = os.path.join('data','mnist_png',name,str(i))
-            for file in os.listdir(path):
-                if file.endswith(".png"):
-                    output_file.write('{},{}\n'.format(os.path.join(os.path.join('mnist_png',name,str(i)), file), str(i)))
\ No newline at end of file
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tempfile
```

Comment:
Ideally, we should be able to completely remove this file and just use `ludwig.datasets.mnist.load` + the split logic in the training script.  Something like:

```
dataset_df = mnist.load()
training_set = dataset_df[dataset_df[SPLIT] == 0]
test_set = dataset_df[dataset_df[SPLIT] == 2]
model.train(training_set=training_set, test_set=test_set, ...)
```

I forgot that `split_dataset_ttv` returns dicts, so we should probably avoid that for now.  Once the Dask PR lands, we can use this method to return Pandas DataFrames.

Reply:
Removed